OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -33,28 +33,8937 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+                                    <msItem xml:id="ms_i1.4.24.1">
+                                        <locus from="98va" to="98vc"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda24Priests">Commemoration of
+                                            the 24 Heavenly Priests</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.24.2">
+                                        <locus from="98vc" to="99rb"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Heda24Azqir"
+                                            >Commemoration of ʾAzqir</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.24.3">
+                                        <locus target="#99rb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda24Qalataw">Commemoration of
+                                            Qālātaw and other saints</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.24.4">
+                                        <locus target="#99rb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda24Diyosqoros">Commemoration
+                                            of two Diyosqoros</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.24.5">
+                                        <locus from="99rb" to="99va"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Heda24Yosef"
+                                            >Commemoration of Yosef</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.24.6">
+                                        <locus target="#99va"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda24SalPriests">Salām-hymn to
+                                            the priests</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.24.7">
+                                        <locus target="#99va"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda24SalAzqir">Salām-hymn to
+                                            ʾAzqir</title>
+                                    </msItem>
+                                </msItem>
+
+                                <msIt
```

Comment:
Yāfq**ǝ**ranna ʾƎgziʾ

Reply:
Again, the manuscript's spelling. I marked it up with `<sic>` now.